nixos/tests/nextcloud/with-postgresql-and-redis.nix

```diff
@@ -46,7 +46,7 @@ in {
       systemd.services.redis = {
         preStart = ''
           mkdir -p /var/run/redis
```

Comment:
`RuntimeDirectory=redis` in the redis module takes care of setting this up at `/run/redis`.